src/test/java/net/masterthought/cucumber/json/EmbeddingTest.java

```diff
@@ -90,6 +90,19 @@ public void getExtension__OnCommonMimeType_ReturnsFileExtension() {
         assertThat(extension).isEqualTo("html");
     }
 
+    @Test
+    public void getExtension__OnCommonMimeTypeWithEncoding_ReturnsFileExtension() {
```

Comment:
double underscore

Reply:
Actually, I used the similar test name pattern as `getExtension__OnCommonMimeType_ReturnsFileExtension` or `getExtension__OnTextMimeType_ReturnsText`.

There are many pre-existing tests with a name that follows the pattern: "getExtension" + double-underscore + "SomeName" + single-underscore + "ReturnsWhatever".

Do you want me to rename my new test methods only? Or to rename the pre-exisiting tests also?

Reply:
interesting approach